_data/cryptocurrencies.yml

```diff
@@ -176,6 +176,13 @@ websites:
       software: Yes
       doc: https://localbitcoins.blogspot.com.au/2013/05/using-two-factor-authentication-on.html
 
+    - name: SatoshiTango
+      url: https://satoshitango.com/
+      img: satoshitango.png
+      tfa: Yes
+      software: Yes
+      doc: https://github.com/2factorauth/twofactorauth/pull/2321#issuecomment-273971890
```

Comment:
I'll go ahead and merge this now, and add the PR to our backlog. This way the site appears right away, and there is a doc listed for it, but we can easily return to add a page to the [notes](https://twofactorauth.org/notes/) section later. 😄